cli/cli.go

```diff
@@ -44,33 +44,35 @@ func init() {
 	}
 
 	// Config vars
-	fs.StringVar(&defaults.RPCHost, "rpc_host", "localhost:50051", "")
```

Comment:
These lines are rearranged only according to the logical groups and order presented in Help